tests/cql/CqlListOperatorsTest.xml

```diff
@@ -385,7 +405,7 @@
 		</test>
 		<test name="IncludedInNullRight">
 			<expression>{'s', 'a', 'm'} included in null</expression>
-			<output>false</output>
+			<output>null</output>
```

Comment:
CQL-to-ELM translates this as `{'s', 'a', 'm'} included in (null as List<Any)`, and according to spec:
> For the list overload, if either argument is null, the result is null.